lib/cli/cli.js

```diff
@@ -12,9 +12,12 @@ var emberCLIVersion      = versionUtils.emberCLIVersion;
 var InstallationChecker  = require('../models/installation-checker');
 
 function CLI(options) {
+  this.name = options.name;
   this.ui = options.ui;
   this.analytics = options.analytics;
   this.testing = options.testing;
+  this.cliRoot = options.cliRoot;
```

Comment:
lets rename this to `root` no need to prefix it.

`cli.cliRoot` vs `cli.root` etc.

(im reviewing now, I may just doit myself quickly, leaving notes here for myself)